Assignment/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="description" content="General Assembly Week 1 Assignment Due 7/24/19">
+        <link rel="stylesheet" href="style/style.css">
+        <title>GA FEWD ASSIGNMENT 1</title>
+    </head>
+    <body>
+        <header>
+            <h1>ANDREW CHANG</h1>
+                <ul>
+                    <li><button onClick="location.href='index.html';">ABOUT ME</button></li>
+                    <li><button  onClick="location.href='other-page/portfolio.html';">PORTFOLIO</button></li>
+                </ul>
+            <hr>
+        </header>
+        <div class="aboutMe">
+            <p>Welcome to my page, my name is Andrew. I am an up and coming developer studying at General Assembly. My hobbies include cooking, League of Legends, and Magic the Gathering. My aspiration is to one day develop programs that combines my love for cooking and video games.</p>
+            <img class="self" src="image/Selfie.jpg" alt="A picture of me being a weeb">
+            <hr class="section">
+            <ul>
+                <li><button class="button" onClick="location.href='https://www.linkedin.com/in/andrew-chang-5b995074/';">linkedIn</button></li>
```

Comment:
no need for onclick anymore when you can use <a href="#" class="button js-button" role="button">Show content</a>
OR use the <button></button> tag